src/components/elements/EditableDiv.js

```diff
@@ -0,0 +1,30 @@
+import "bootstrap-icons/font/bootstrap-icons.css";
+import classes from "./EditableDiv.module.css";
+
+function EditableDiv(props) {
+  function editPostHandler(postData) {
+    fetch(
+      `https://friendface-react-default-rtdb.europe-west1.firebasedatabase.app/posts/${props.data.id}.json`,
+      {
+        method: "PATCH",
+        body: JSON.stringify(postData),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
+    // TODO: Feedback to the user that the post change was successful .then() ?
```

Comment:
Why can't this be done as a part of this task/PR? Unless it is a large ordeal or that we're pressed by deadlines, I think we should finish the feature properly